test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -224,6 +224,9 @@ Function sys.bbf_binary_varbinary_cmp(sys.bbf_binary,sys.bbf_varbinary)
 Function sys.bbf_get_context_info()
 Function sys.bbf_get_current_physical_schema_name(text)
 Function sys.bbf_is_shared_schema(text)
+Function sys.bbf_log(double precision)
```

Comment:
Please add upgrade tests for bbf_log functions as well (if it's public accessible)

Reply:
Tests for bbf_log functions are at the end of verify.sql